content/influxdb/v2.4/get-started/setup.md

```diff
@@ -0,0 +1,332 @@
+[influx CLI](#)
+[InfluxDB API](#)
+{{% /tabs %}}
+
+{{% tab-content %}}
+<!------------------------------ BEGIN UI CONTENT ----------------------------->
+
+1.  Visit
+    {{% oss-only %}}[localhost:8086](http://localhost:8086){{% /oss-only %}}
+    {{% cloud-only %}}[cloud2.influxdata.com](https://cloud2.influxdata.com){{% /cloud-only %}}
+    in a browser to log in and access the InfluxDB UI.
+
+2.  Navigate to **Load Data** > **API Tokens** using the left navigation bar.
+3.  Click **+ {{% caps %}}Generate API token{{% /caps %}}** and select
+    **All Access API Token**.
+4.  Enter a description for the API token and click **{{< icon "check" >}} {{% caps %}}Save{{% /caps %}}**.
+5.  Copy the generated token and store it for safe keeping.
+
+<!------------------------------- END UI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!---------------------------- BEGIN CLI CONTENT ----------------------------->
+
+1.  If you haven't already, [download, install, and configure the `influx` CLI](/influxdb/v2.4/tools/influx-cli/).
+2.  Use the [`influx auth create` command](/influxdb/v2.4/reference/cli/influx/auth/create/)
+    to create an all access token.
+    
+    **Provide the following**:
+
+    - `--all-access` flag
+    - Operator token to authorize the request
+
+    {{< cli/influx-creds-note >}}
+
+    ```sh
+    influx auth create --all-access
+    ```
+
+3.  Copy the generated token and store it for safe keeping.
+
+<!------------------------------ END CLI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!----------------------------- BEGIN API CONTENT ----------------------------->
+
+Send a request to the InfluxDB API `/api/v2/authorizations` endpoint using the `POST` request method.
+
+{{< api-endpoint endpoint="http://localhost:8086/api/v2/authorizations" method="post" >}}
+
+Include the following with your request:
+
+- **Headers**:
+  - **Authorization**: Token <INFLUX_OPERATOR_TOKEN>
+  - **Content-Type**: application/json
+- **Request body**: JSON body with the following properties:
+  - **status**: "active" or "inactive"
+  - **description**: API token description
+  - **orgID**: [InfluxDB organization ID](/influxdb/v2.4/organizations/view-orgs/#view-your-organization-id)
+  - **permissions**: Array of objects where each object represents permissions
+    for an InfluxDB resource. Each permission contains the following properties:
+      - **action**: "read" or "write"
+      - **resource**: JSON object that represents the InfluxDB resource to grant
+        permission to. Each resource contains at least the following properties:
+          - **orgID**: [InfluxDB organization ID](/influxdb/v2.4/organizations/view-orgs/#view-your-organization-id)
+          - **type**: Resource type.
+            _For information about what InfluxDB resource types exist, use the
+            [`/api/v2/resources` endpoint](/influxdb/v2.4/api/#operation/GetResources)._
+
```

Comment:
The following example shows how to use cURL with the InfluxDB API to generate an all access token: